script.js

```diff
@@ -0,0 +1,49 @@
+// Testing on one cell clicking to change to
+// X and adding another class to cell
+
+var example = document.getElementById('c3')
+
+function exampleBox () {
+    if (example.innerHTML =="") {
+    example.innerHTML = 'X';
+    example.className += ' blue'
+    example.style.color = "blue";
+  }
+
+    else if (example.innerHTML == 'X') {
+      example.innerHTML = 'O';
+      example.className += ' red'
+      example.style.color = "red";
+    }
+}
+example.addEventListener("click", exampleBox, false);
```

Comment:
you can dynamically handle these click events by using `event.target` instead of `example` in your `exampleBox` function above - https://developer.mozilla.org/en-US/docs/Web/API/Event/target